src/beamlime/logging/loggers.py

```diff
@@ -3,24 +3,21 @@
 
 import logging
 import sys
-from contextlib import ContextDecorator
+from contextlib import contextmanager
 from typing import Union
 
 from .sources import find_caller
 
 
-class hold_logging(ContextDecorator):
-    def __enter__(self):
-        from . import _lock
+@contextmanager
+def hold_logging() -> None:
```

Comment:
Is this for logging multiple messages that form an atomic unit, i.e., should not get interleaved with other messages?